config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 # This is a sig-release-master-blocking job.
-- cron: '1 22 * * 6' # Run at 14:01PST on Sat (22:01 UTC on Sat)
+- cron: '1 02 * * 2,3,4,5,6' # Run at 18:01PST on Mon-Fri (02:01 UTC on Tue-Sat)
```

Comment:
We need to adjust timeouts for both performance and correctness so that they will for sure not overlap with each other.

Reply:
What about 20h for performance and 3h for correctness (and 1h for buffer between the two)? So something like:
- performance starts 8:00 with 20h timeout
- correctness starts 4:30 with 3h timeout

Any thoughts?

Reply:
We can do less than 20h - it currently take between 13 and 14 hours.
So let's do:
18h for performance
1h break
4h for correctness
1h break

we can adjust later if needed.

Reply:
Done.